starcheck/src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -65,6 +65,39 @@ def _get_agasc_stars(ra, dec, roll, radius, date, agasc_file):
 
     return stars_dict
 
+import numpy as np
+import mica.stats.acq_stats
+import mica.stats.guide_stats
+
+ACQS = mica.stats.acq_stats.get_stats()
+GUIDES = mica.stats.guide_stats.get_stats()
+
+
+def get_mica_stats(agasc_id, time):
+
+    acqs = ACQS[ACQS['agasc_id'] == int(agasc_id)]
+    time = float(time)
+    acq_5y = acqs[(acqs['guide_tstart'] >= (time - (5 * 365 * 86400)))
+                 & (acqs['guide_tstart'] <= time)]
+    ok_5y = (acq_5y['img_func'] == 'star') & (~acq_5y['ion_rad']) & (~acq_5y['sat_pix'])
+
+    guides = GUIDES[GUIDES['agasc_id'] == int(agasc_id)]
+
+    mags = acqs['mag_obs'][acqs['mag_obs'] != 0].tolist()
```

Comment:
`np.concatenate([x, y, ..])` is a good function for your toolbox and works well here.

Reply:
I started with np.concatenate.  I didn't seem to work if either x or y was empty.  I'll go back to its docs.

Reply:
I must have just has something else wrong when I blamed np.concatenate